example/grpc-web/lib/app.dart

```diff
@@ -61,7 +61,7 @@ class EchoApp {
         ..append(HTMLSpanElement()
           ..classList.add('label')
           ..classList.addAll(cssClass)
-          ..text = message)));
+          ..textContent = message)));
```

Comment:
Should I keep this? CI was failing here, even if this has nothing to do with this PR

Reply:
This is fixed in #765

Reply:
Okk, I'll revert as soon as 765 is merged.